layouts/partials/index_abcdj.html

```diff
@@ -179,7 +187,41 @@ <h3><a href="https://github.com/Biomarker-Development-at-INM7">GitHub</a></h3>
                     <img src="../pics/jtrack_logo.svg" alt="">
                 </div>
             </div>
+            <a href="#section5" class="profile-scroll push"><i class="fas fa-chevron-down"></i></a>
```

Comment:
FTR, there also is a support channel for Julearn: https://matrix.to/#/#ml:inm7.de